src/ArmTemplates/Creator/Models/CreatorConfiguration.cs

```diff
@@ -2,6 +2,7 @@
 using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.TemplateModels;
 using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.Templates.Abstractions;
 using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.Templates.Logger;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Extractor.Models;
 
 namespace Microsoft.Azure.Management.ApiManagement.ArmTemplates.Creator.Models
```

Comment:
Can all of these classes/enums get their own file? 
Also, are they used as DTOs or why did you not apply the naming rules to this file?

Reply:
I didn't refactor Creator still (it is upcoming). I dont want to change like everything right now, because I will definitely forget about something I have changed and it will be a headache later (I think we will get some bugs after refactoring is over, but hope we can reduce it's amount if we refactor things one by one).

So I would prefer to leave as it is for now.